### DISCUSSION TITLE
Keep position on page when clicking back button

### DISCUSSION THREAD

**Discussion by croganm** [dailydotdev/daily#823] [Feature Requests]
Please please please, the most annoying thing about using daily.dev is when I click the supplied back button within the app, and it takes me to the very top of home page 🙄. I would love the ability to continue from the article I just left rather than have to scroll all the way back down. I'm on iOS and have it added to my home screen.

**Comment by idoshamun** [comment]
This is an annoying one, for sure. 
We have an active discussion here https://github.com/dailydotdev/daily/discussions/728.
We'll do my best to prioritize it.
